### PR TITLE
Show average rounded to 2 decimal on hover

### PR DESCRIPTION
**Pull request by LiJu09**
I don't know if it's useful, but it can be especially for users because it's a small amount, and maybe 1 decimal would be enough.
I know it's doesn't work on touchscreens, but if this will be approved I can add it.
Or are there any plans for adding more features such as graphs, where it would be rounded to at least 1 decimal place?

### CODE REVIEW

**Comment by Adminiuga** [comment]
Some people may want 0.01 steps

File: site/src/analytics-average.ts [hunk 1]
```diff
@@ -13,19 +13,31 @@ export class AnalyticsAverage extends LitElement {
     return html`<div class="grid">
         <div class="metric">
           <span>Average integrations</span>
-          <span>${Math.round(this.lastDataEntry.avg_integrations)}</span>
+          <span
+            title="${Math.round(this.lastDataEntry.avg_integrations * 100) /
```
Comment (ludeeus) [inline site/src/analytics-average.ts:17]: Use  `.title=${}` instead of `title="${}"`

**Review by ludeeus — CHANGES_REQUESTED** [review]
Let's create const above the return statement and reference those

```js

const integrations = this.lastDataEntry.avg_integrations.toFixed(2);
...

return html...
...
<div class="metric">
  <span>Average integrations</span>
  <span .title=${integrations}>
    ${Math.round(Number(integrations))}
  </span>
</div>
```

**Review by ludeeus — APPROVED** [review]
As for your questions about the future, there are currently no plans and the history part of the dataset does not currently include this, so there is nothing to graph really.

**Comment by LiJu09** [comment]
And something like this for touchscreen it's necessary?

**Comment by ludeeus** [comment]
No graphs are not needed for touch

**Comment by LiJu09** [comment]
I mean that pop up on hover.

**Comment by ludeeus** [comment]
<https://material.io/components/tooltips#behavior> Have a look here.

**Pull request merged**